rpcd/playbooks/roles/kibana/tasks/kibana_post_install.yml

```diff
@@ -40,8 +74,8 @@
   template:
     src: "{{ item }}"
     dest: "/etc/apache2/sites-available/{{ item }}"
-    owner: "{{ kibana_system_user }}"
-    group: "{{ kibana_system_group }}"
+    owner: "root"
+    group: "root"
```

Comment:
I'm kinda inclined to agree that these probably shouldn't be root, though being root isn't the worst thing for an apache config file.